package.json

```diff
@@ -1,125 +1,24 @@
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "author": "",
   "scripts": {
-    "build": "ember build --environment=production",
-    "lint": "concurrently \"pnpm:lint:*(!fix)\" --names \"lint:\"",
-    "lint:css": "stylelint \"**/*.css\"",
-    "lint:css:fix": "concurrently \"pnpm:lint:css -- --fix\"",
-    "lint:fix": "concurrently \"pnpm:lint:*:fix\" --names \"fix:\"",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
-    "test": "concurrently \"pnpm:lint\" \"pnpm:test:*\" --names \"lint,test:\"",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
-  },
-  "dependencies": {
-    "@babel/core": "^7.25.2",
-    "ember-cli-babel": "^8.2.0"
-  },
+    "build": "pnpm --filter @ember/render-modifiers build",
+    "lint": "pnpm --filter '*' lint",
+    "lint:fix": "pnpm --filter '*' lint:fix",
+    "prepare": "pnpm build",
+    "start": "concurrently 'pnpm:start:*' --restart-after 5000 --prefixColors auto",
+    "start:addon": "pnpm --filter @ember/render-modifiers start --no-watch.clearScreen",
+    "start:test-app": "pnpm --filter test-app start",
+    "test": "pnpm --filter '*' test",
+    "test:ember": "pnpm --filter '*' test:ember"
+  },
+  "packageManager": "pnpm@10.0.0",
   "devDependencies": {
-    "@babel/eslint-parser": "^7.25.1",
-    "@babel/plugin-proposal-decorators": "^7.24.7",
-    "@ember/optional-features": "^2.1.0",
-    "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^3.3.1",
-    "@embroider/test-setup": "^4.0.0",
-    "@embroider/util": "^1.0.0",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "@glint/template": "^1.0.2",
-    "concurrently": "^8.2.2",
-    "ember-auto-import": "^2.8.1",
-    "ember-cli": "~5.12.0",
-    "ember-cli-clean-css": "^3.0.0",
-    "ember-cli-dependency-checker": "^3.3.2",
-    "ember-cli-htmlbars": "^6.3.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-terser": "^4.0.2",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^8.2.3",
-    "ember-qunit": "^8.1.0",
-    "ember-resolver": "^12.0.1",
-    "ember-source": "~5.12.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^6.0.0",
-    "ember-try": "^3.0.0",
-    "eslint": "^8.57.1",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-ember": "^12.2.1",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-prettier": "^5.2.1",
-    "eslint-plugin-qunit": "^8.1.2",
-    "loader.js": "^4.7.0",
-    "prettier": "^3.3.3",
-    "qunit": "^2.22.0",
-    "qunit-dom": "^3.2.1",
-    "release-it": "^14.11.6",
-    "release-it-lerna-changelog": "^3.1.0",
-    "stylelint": "^15.11.0",
-    "stylelint-config-standard": "^34.0.0",
-    "stylelint-prettier": "^4.1.0",
-    "webpack": "^5.95.0"
-  },
-  "peerDependencies": {
-    "@glint/template": "^1.0.2",
```

Comment:
This peerDep is probably still needed (together with the peerDependenciesMeta config).